test/src/main/java/org/apache/accumulo/test/functional/ZooMutatorIT.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.test.functional;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.accumulo.core.client.Accumulo;
+import org.apache.accumulo.core.clientImpl.ClientContext;
+import org.apache.accumulo.core.conf.Property;
+import org.apache.accumulo.fate.zookeeper.ZooReaderWriter;
+import org.apache.accumulo.harness.AccumuloClusterHarness;
+import org.apache.commons.codec.digest.DigestUtils;
+import org.junit.Test;
+
+public class ZooMutatorIT extends AccumuloClusterHarness {
+  /**
+   * A simple stress test that looks for race conditions in
+   * {@link ZooReaderWriter#mutateOrCreate(String, byte[], org.apache.accumulo.fate.zookeeper.ZooReaderWriter.Mutator)}
+   */
+  @Test
+  public void concurrentMutatorTest() throws Exception {
+    try (var client = Accumulo.newClient().from(getClientProps()).build()) {
+
+      @SuppressWarnings("resource")
+      var context = (ClientContext) client;
+      var secret = cluster.getSiteConfiguration().get(Property.INSTANCE_SECRET);
```

Comment:
Some of these uses of `var` are a bit questionable, since they don't make the line shorter or more readable, nor are they used when the variable's type is obvious. I would use the actual type in many of these.

```suggestion
      String secret = cluster.getSiteConfiguration().get(Property.INSTANCE_SECRET);
```

Reply:
> Some of these uses of var are a bit questionable, since they don't make the line shorter or more readable

I agree.  It was faster to use them when writing the code, but it is less readable.